pkg/ctl/scale/nodegroup.go

```diff
@@ -1,11 +1,10 @@
 package scale
 
 import (
-	"github.com/weaveworks/eksctl/pkg/actions/nodegroup"
-
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/weaveworks/eksctl/pkg/actions/nodegroup"
```

Comment:
:shef'skiss:

Reply:
I started noticing this after you mentioned it a couple of weeks ago! :D